source/Random.cpp

```diff
@@ -104,12 +111,53 @@ uint32_t Random::Binomial(uint32_t t, double p)
 
 
 
-// Get a normally distributed number (mean = 0, sigma= 1).
-double Random::Normal()
+// Get a normally distributed number (mean = 0, sigma= 1) using std::normal_distribution.
+double Random::StdNormal()
 {
 	normal_distribution<double> normal;
 #ifndef __linux__
 	lock_guard<mutex> lock(workaroundMutex);
 #endif
 	return normal(gen);
 }
+
+
+
+// Get a normally distributed number (mean = 0, sigma= 1) using std::normal_distribution.
+double Random::StdNormal(double mean, double sigma)
+{
+	normal_distribution<double> normal{mean, sigma};
```

Comment:
```suggestion
	normal_distribution<double> normal(mean, sigma);
```

Reply:
@tehhowch Curious as to why. The vast majority of the code uses `()` to initialize objects.

Reply:
We should prefer curly braces for initialization unless we are avoiding an initializer list constructor (e.g. filling a vector with copies of a value).

It's not worth the churn to go and change the existing code, but new code should use it.

Reply:
I mean I just used the braces because that's what I thought the documentation (as far as there is any) said was correct if you didn't want to get into `distribution = Distribution(params)`.